Index: framework-core/src/main/java/cn/vansky/framework/core/orm/mybatis/plugin/page/SQLHelp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- framework-core/src/main/java/cn/vansky/framework/core/orm/mybatis/plugin/page/SQLHelp.java	(revision cc5b86fa7f1d48bd40248eb8fdfc2c34bfef6f07)
+++ framework-core/src/main/java/cn/vansky/framework/core/orm/mybatis/plugin/page/SQLHelp.java	(revision )
@@ -238,8 +238,8 @@
         //order拼接
         customCallback.prepareOrder(sb,parameter);
         //分页
-        originalSql= generateRealPageSql(sb.toString(),parameter,dialect);
-        return originalSql;
+        customCallback.setPageable(sb,parameter,dialect);
+        return sb.toString();
     }
 
     /**
